config/tsa/tsa/300-tsa.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tsa
+  namespace: tsa-system
+---
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  namespace: tsa-system
+  name: tsa
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/min-scale: "1"
+    spec:
+      serviceAccountName: tsa
+      containers:
+      - name: tsa
+        image: ko://github.com/sigstore/timestamp-authority/cmd/timestamp-server/
+        ports:
+        - containerPort: 3000
+        args: [
+          "serve",
+          "--timestamp-signer=file",
+          "--file-signer-key-path=/var/run/tsa-secrets/signing-secret",
```

Comment:
lemme know if there's any issues! i haven't tested with the file signer, but this was just a copy paste from rekor

Reply:
No issues, the e2e test that I added to github actions works just fine (fairly straight up copy from the README.md).
https://github.com/sigstore/scaffolding/pull/464/files#diff-6fc761a0f69ff1560f6fb38a4bc8c509cb4d9c68df250223f765ca6564521119R234